cns/cniconflist/generator.go

```diff
@@ -0,0 +1,40 @@
+package cniconflist
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/Azure/azure-container-networking/cns/configuration"
+)
+
+type Generator struct {
+	writer   io.WriteCloser
+	scenario configuration.CNIConflistScenario
+}
+
+func (g *Generator) Close() error {
+	if err := g.writer.Close(); err != nil {
+		return fmt.Errorf("error closing generator: %w", err)
```

Comment:
`errors.Wrap` is used in CNS, typically.

Reply:
changed to `errors.Wrap`